### PR TITLE
create debug_shell_enable config to enable debug shell

### PR DESCRIPTION
**Pull request by VenkatCisco**
Signed-off-by: Venkat Garigipati <venkatg@cisco.com>

**What I did**
Enable Debug shell config to enable debug shell. 

**Why I did it**
Provision for Debug shell

**How I verified it**

**Details if related**

### CODE REVIEW

File: orchagent/switchorch.cpp [hunk 1]
```diff
@@ -26,6 +26,7 @@ const map<string, sai_switch_attr_t> switch_attribute_map =
     {"ecmp_hash_seed",                      SAI_SWITCH_ATTR_ECMP_DEFAULT_HASH_SEED},
     {"lag_hash_seed",                       SAI_SWITCH_ATTR_LAG_DEFAULT_HASH_SEED},
     {"fdb_aging_time",                      SAI_SWITCH_ATTR_FDB_AGING_TIME},
+    {"debug_shell_enable",                  SAI_SWITCH_ATTR_SWITCH_SHELL_ENABLE},
```
Comment (prsunny) [inline orchagent/switchorch.cpp:29]: once enabled, how do you disable?

Comment (VenkatCisco) [reply orchagent/switchorch.cpp:29]: This is a create and set event as per SAI spec.[ ](https://github.com/opencomputeproject/SAI/blob/master/inc/saiswitch.h#L1671)

Comment (VenkatCisco) [reply orchagent/switchorch.cpp:29]: https://github.com/opencomputeproject/SAI/blob/master/inc/saiswitch.h#L1671

Comment (rameshsanth) [reply orchagent/switchorch.cpp:29]: By setting 1 to 0 in below json, the debug shell can be disabled
```
[
    {
        "SWITCH_TABLE:switch": {
            "debug_shell_enable": "1"
        },
        "OP": "SET"
    }
]
```

Comment (prsunny) [reply orchagent/switchorch.cpp:29]: ok

**Review by prsunny — APPROVED** [review]

**Comment by VenkatCisco** [comment]
Thanks for the merge. Can you help with double commit this fix into 202012 branch as well ?.

**Pull request merged**